Adding managed volume actions

jasonburrell opened the pull request.

chriswahl commented:
@jasonburrell Did the PR template not load? We're looking for a completed PR and associated Issue for code merges.

jasonburrell commented:
Not sure what you mean by that. I added the files that I added and created the PR, you should of been able to see the code changes..

Jason Burrell
360.301.5764| @rubrikInc <https://twitter.com/rubrikInc> | @jasonburrell2 <https://twitter.com/jasonburrell2>
> On Dec 4, 2017, at 3:40 PM, Chris Wahl <notifications@github.com> wrote:
> 
> @jasonburrell <https://github.com/jasonburrell> Did the PR template not load? We're looking for a completed PR and associated Issue for code merges.
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub <https://github.com/rubrikinc/PowerShell-Module/pull/147#issuecomment-349144658>, or mute the thread <https://github.com/notifications/unsubscribe-auth/ASZd1SpEwKvEDwagaR3-qV3YUIXVQMnaks5s9ILjgaJpZM4QzbUT>.
>

chriswahl commented:
That's weird. I mean, yes, I can see your file changes and commits, but the PR usually loads with a form template for you to fill out that asks about the code submission.

You should see [this template](https://github.com/rubrikinc/PowerShell-Module/blob/master/.github/PULL_REQUEST_TEMPLATE.md) in GitHub to help communicate all of the required information for the change.

If it didn't load, just replace your PR description with the markdown details and fill in the answers.
